templates/dhcpd.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=DHCPv4 Server Daemon
+Documentation=man:dhcpd(8) man:dhcpd.conf(5)
+Wants=network-online.target
+After=network-online.target
+After=time-sync.target
+
+[Service]
+Type=notify
+ExecStart=/usr/sbin/dhcpd -f -cf /etc/dhcp/dhcpd.conf -user dhcpd -group dhcpd --no-pid <%= Array(@dhcp_interfaces).join(' ') %>
```

Comment:
$dhcp_dir should be used instead of hardcoding the path to the config file. At least on arch it is /etc/dhcpd.conf

Reply:
Good point, and adding in Arch spec also 👍